sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -168,6 +168,12 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
       return
     }
     lifecycleState = LifecycleState.STATE_DESTROYED
+    // flush the queued events before destroy to avoid lost events
```

Comment:
can't say I fully understand what is 'lost event' here; if I understand correctly `flush` here clears the queue but how bad is it if we have that 'lost event'? could it be a memory leak?

Reply:
some of the telemetry events will be lost, which is not desirable, but no memory leak 😅